packages/garbo/src/tasks/barfTurn.ts

```diff
@@ -355,7 +355,7 @@ const NonBarfTurnTasks: AlternateTask[] = [
       }
       ChestMimic.differentiate(globalOptions.target);
     },
-    outfit: () => embezzlerOutfit({ familiar: $familiar`Chest Mimic` }),
+    outfit: () => meatTargetOutfit({ familiar: $familiar`Chest Mimic` }),
```

Comment:
Again, is it worth adding in something in the case of freefight target vs meat target vs [other] target?